lib/mongoid/locker.rb

```diff
@@ -15,14 +15,14 @@ module ClassMethods
       #
       # @return [Mongoid::Criteria]
       def locked
-        where locked_until_field.gt => Time.now
+        where locked_until_field.gt => Time.now.utc
```

Comment:
Can we get a 'now' method extracted so all of these references are deduplicated?

Reply:
Do you suggest put `Time.now.utc` into `now` method and  use it like this?
```ruby
where locked_until_field.gt => Mongoid::Locker.now
```

Reply:
No worries, I will send a separate pull request.